test/RedisOutputCachingMiddleWareFunctionalTests/RedisOutputCachingMiddleWareFunctionalTests.cs

```diff
@@ -0,0 +1,91 @@
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Builder;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using RedisOutputCachingMiddleware;
+using System;
+using Microsoft.AspNetCore.Http;
+using System.Threading;
+using Microsoft.Web.Redis.FunctionalTests;
+
+namespace RedisOutputCachingMiddleWare.FunctionalTests
+{
+    public class RedisOutputCachingMiddleWareFunctionalTests
+    {
+        private string GetUnixTimeSeconds()
+        {
+            return DateTimeOffset.UtcNow.ToUnixTimeSeconds().ToString();
+        }
+
+        [Fact]
+        public void TestWithoutCache()
+        {
+            Assert.True(ResponseIsCurrent().Result);
```

Comment:
Avoid using `.Result` if at all possible. It should be possible to convert these test methods to `async`